### ISSUE TITLE
Adding composer.json and submitting to packagist

### ISSUE THREAD

**Issue by chewett** [benpickles/peity#86]
Would you consider adding a composer.json file so that this library can be included in a composer based PHP project easily?

If wanted I can create the composer file and submit a pull request.

Currently there is a clone of the repo in packagist (https://packagist.org/packages/grimmlink/peity) but it would be best to include your repo rather than someones clone

Thanks

**Comment by benpickles** [comment]
Sure, let's give it a try.

Good idea, you create the `composer.json` and we can integrate it together.

**Comment by benpickles** [comment]
Looks like this might be perfect timing: https://github.com/blog/2247-improving-collaboration-with-forks

**Comment by chewett** [comment]
I have made a pull request in: https://github.com/benpickles/peity/pull/87

Once this is merged into the repo you will need to log into packagist (https://packagist.org/) with your github credentials (oauth I believe) and submit the repo for inclusion.

Once submission is done it will index your repo and make every tagged version containing the composer file available for easy inclusion. Therefore I recommend, once the composer file is merged in, to release another tagged minor version so that its immediately accessible via packagist.

Thanks
Chris

**Comment by benpickles** [comment]
Great, I'll take a look.
